src/Microsoft.AspNet.Mvc.Razor.Host/MvcRazorHost.cs

```diff
@@ -42,6 +42,10 @@ public class MvcRazorHost : RazorEngineHost, IMvcRazorHost
             {
                 LookupText = "Microsoft.AspNet.Mvc.Razor.TagHelpers.UrlResolutionTagHelper, Microsoft.AspNet.Mvc.Razor"
             },
+            new AddTagHelperChunk
```

Comment:
Spoke to @Eilon - instead of doing this, move it to `Microsoft.AspNet.Mvc.TagHelpers`. Users will import it via `_ViewImports`.